mage/query-modules/python/xml_module.md

```diff
@@ -0,0 +1,614 @@
+---
+id: xml_module
+title: xml_module
+sidebar_label: xml_module
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+  <span
+    style={{
+      backgroundColor: color,
+      borderRadius: '2px',
+      color: '#fff',
+      padding: '0.2rem',
+    }}>
+    {children}
+  </span>
+);
+
+The XML module enhances graph database capabilities by providing support for loading and parsing XML data.
+
+[![docs-source](https://img.shields.io/badge/source-xml_module-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/blob/main/python/xml_module.py)
+
+
+| Trait               | Value                                                 |
+| ------------------- | ----------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**util**</Highlight>  |
+| **Implementation**  | <Highlight color="#FB6E00">**Python**</Highlight>     |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight> |
+
+
+
+## Functions
+
+### `parse(xml_input, simple, path)`
+
+Parses an XML string or file into a map. Since this is a function and not a procedure, it can be used inline in your Cypher queries.
+
+#### Input:
+
+- `xml_input: string` ➡ input XML string.
+- `simple: bool (default = false)` ➡ bool used for specifying whether simple mode should be used.
```

Comment:
what is a simple mode?

Reply:
that heading is called simple configuration, should this explanation also use the word configuration so i can connect these two